test_xpartition.py

```diff
@@ -317,7 +338,7 @@ def __call__(self, dsk, keys, **kwargs):
 
 
 @pytest.mark.parametrize(
-    ("collect_variable_writes", "expected_computes"), [(False, 6), (True, 3)]
+    ("collect_variable_writes", "expected_computes"), [(False, 9), (True, 3)]
```

Comment:
This changes because we now include chunked coordinates on each DataArray returned by `_construct_dataarray`.